core/src/main/java/com/bankbazaar/tripmanager/manager/RegisterManager.java

```diff
@@ -0,0 +1,38 @@
+package com.bankbazaar.tripmanager.manager;
+
+import com.bankbazaar.tripmanager.model.Users;
+import com.bankbazaar.tripmanager.repository.UsersRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCrypt;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+@Service
+public class RegisterManager {
```

Comment:
The managers should not be basis the functions, but basis the entity.

For all User related task it should be in UserManager. There should not be anything like RegisterManager

Reply:
register controller and manager removed